test/NosCore.Tests/PacketHandlerTests/MJoinPacketHandlerTests.cs

```diff
@@ -169,5 +169,79 @@ public void Join()
 
             Assert.IsTrue(_session.Character.MapInstance.Map.MapId == 20001);
         }
+
+        [TestMethod]
+        public void JoinPrivate()
+        {
+            var mjoinPacket = new MJoinPacket
+            {
+                VisualId = _targetSession.Character.CharacterId,
+                Type = VisualType.Player
+            };
+            _connectedAccountHttpClient.Setup(s => s.GetCharacter(_targetSession.Character.CharacterId, null))
+                .Returns((new ServerConfiguration(),
+                    new ConnectedAccount
+                    {
+                        ChannelId = 1, ConnectedCharacter = new Character { Id = _targetSession.Character.CharacterId }
+                    }));
+            _connectedAccountHttpClient.Setup(s => s.GetCharacter(_session.Character.CharacterId, null))
+                .Returns((new ServerConfiguration(),
+                    new ConnectedAccount
+                    { ChannelId = 1, ConnectedCharacter = new Character { Id = _session.Character.CharacterId } }));
+            _friendHttpClient.Setup(s => s.GetListFriends(It.IsAny<long>())).Returns(new List<CharacterRelationStatus>
+            {
+                new CharacterRelationStatus
+                {
+                    CharacterId = _targetSession.Character.CharacterId,
+                    IsConnected = true,
+                    CharacterName = _targetSession.Character.Name,
+                    RelationType = CharacterRelationType.Friend
+                }
+            });
+            _minilandProvider.Setup(s => s.GetMiniland(It.IsAny<long>())).Returns(new Miniland
+            { MapInstanceId = TestHelpers.Instance.MinilandId, State = MinilandState.Private });
+            _mjoinPacketHandler.Execute(mjoinPacket, _session);
+
+            var lastpacket = (InfoPacket)_session.LastPackets.FirstOrDefault(s => s is InfoPacket);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'lastpacket'.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4513002)